app/assets/javascripts/application/main.js

```diff
@@ -20,12 +20,3 @@
     }
   }
 }());
-
-webshims.setOptions('basePath', '/webshims/shims/')
```

Comment:
now that we're deleting this, we can also get rid of `lib/tasks/webshims_asset_compile.rake`, L78 (`RUN bundle exec rake webshims:update_public`), and the webshims-rails gem!